src/_guides/language/language-tour.md

```diff
@@ -3033,6 +3033,51 @@ class Point {
 }
 ```
 
+#### Operators {#_operators}
+
+Operators are instance methods with special names.
+Dart allows you to define operators with the following names:
+
+`<`  | `+`  | `|`  | `[]`
+`>`  | `/`  | `^`  | `[]=`
+`<=` | `~/` | `&`  | `~`
+`>=` | `*`  | `<<` | `==`
+`–`  | `%`  | `>>`
```

Comment:
It might be worth double checking this list of operators against the language spec, ... just in case it has changed.